deploy/test/test_cases/TEST_ID_10_SECRETS_DESTINATION_not_exist.sh

```diff
@@ -9,7 +9,8 @@ echo "Create test-env pod. SECRETS_DESTINATION is with invalid value 'incorrect_
 export SECRETS_DESTINATION_KEY_VALUE=$KEY_VALUE_NOT_EXIST
 deploy_init_env
 
-pod_name=$(cli_get_pods_test_env | awk '{print $1}')
+$cli_with_timeout "get pods --namespace=$APP_NAMESPACE_NAME --selector app=test-env --no-headers | wc -l | tr -d ' ' | grep '^1$'"
+pod_name=$($cli_with_timeout "get pods --namespace=$APP_NAMESPACE_NAME --selector app=test-env --no-headers" | awk '{print $1}')
```

Comment:
We have to run it twice ? Did we try to run the second line only? if it didnt work then why we need to run the "get pods command" twice with timeout.. 
The second one can be without timeout.